zulipterminal/config/themes.py

```diff
@@ -1,64 +1,9 @@
 from typing import Dict, List, Optional, Tuple
 
+from zulipterminal.themes.Common import REQUIRED_STYLES
```

Comment:
Interesting, having the helper function in color.py would be cleaner I think.

I've used a capital letter because sorting in the directory tree places it to the top.

Is a rename to REQUIRED_STYLE also not required? as it is like a constant definition? and also compares with STYLE